common/remote/sockfile.cpp

```diff
@@ -6330,10 +6351,11 @@ class CRemoteFileServer : implements IRemoteFileServer, public CInterface
                         }
                         pastFirstRow = true;
                     }
+                    while (reply.length() < replyLimit);
```

Comment:
do we ever stream out partial records?

Reply:
no.

Reply:
Do we cope with case where single record is larger than the limit?

Reply:
Yes in the sense that at least 1 record is serialized regardless, so if 1 rec is > than limit, then the returned size will be approx whatever that 1 rec size is.

Reply:
Ok - caller / API documentation will need to be aware that the passed-in size limit may be exceeded in this case then.